src/extension.ts

```diff
@@ -0,0 +1,48 @@
+import * as vscode from 'vscode';
+
+export function activate(context: vscode.ExtensionContext) {
+	
+	// Use the console to output diagnostic information (console.log) and errors (console.error)
+	// This line of code will only be executed once when your extension is activated
+	console.log('Congratulations, your extension "SM213" is now active!');
+
+	// The command has been defined in the package.json file
+	// Now provide the implementation of the command with registerCommand
+	// The commandId parameter must match the command field in package.json
+	let disposable = vscode.commands.registerCommand('sm213-assembly-syntax-highlighting.credit', () => {
+		// The code you place here will be executed every time your command is executed
+		// Display a message box to the user
+		vscode.window.showInformationMessage('0000 - 2021 Humanity!');
```

Comment:
What does this do 😅 ?

Reply:
Ah..ha... We can always change the Easter egg to something else or remove it😅.

Reply:
Ohhh I couldn't figure out how to trigger it at first lol -- you run `sm213-assembly-syntax-highlighting.credit` in the command palette 🤦 . I like it, let's keep it!

<img width="500" alt="Screen Shot 2021-09-28 at 11 13 49 PM" src="https://user-images.githubusercontent.com/63937200/135213047-be803e22-d7e8-4e3b-8285-21ae78606ce5.png">